layouts/partials/posts/pagination.html

```diff
@@ -1,18 +1,119 @@
-<!-- Footer -->
-{{ if gt .paging.TotalPages 1 }}
+{{ $paginator := .paginator }}
+{{ $outer := .outer}}
+
+<!-- Thanks to https://glennmccomb.com/articles/how-to-build-custom-hugo-pagination/ -->
+
+<!-- Show first and last link-->
+{{ $show_first_last := $outer.Site.Params.Posts.showFirstLast | default false }}
```

Comment:
Testing this locally I can only get the `showFirstLast` value to return from the config file if I change the casing to:

```
{{ $show_first_last := $outer.Site.Params.posts.showfirstlast | default false }}
```

I'm not sure why this is?

Reply:
This is very strange. I am using Hugo version 0.54.0 

Could you try outer.Site.params.posts.showFirstLast since they are lower case in the settings file.

If that does not work it may be a bug in Hugo? On my side it shows up in all those cases.

Reply:
Upgrading to `0.54.0` fixed this for me, however now none of the translations are appearing..

This is happening on `master` too, so is an existing issue, but it'll need fixing before this can be merged and the min version increased. I'm currently looking into it.

Reply:
I found a fix by supplying the theme directory along with the theme name in the config. I also changed the other config params to a consistent spelling